Formula/llvm.rb

```diff
@@ -275,10 +279,59 @@ def caveats
     # related to https://github.com/Homebrew/legacy-homebrew/issues/47149
     system "#{bin}/clang++", "-v",
            "-std=c++11", "-stdlib=libc++", "test.cpp", "-o", "testlibc++",
-           "-Wl,-rpath,#{opt_lib}"
+           "-L#{opt_lib}", "-Wl,-rpath,#{opt_lib}"
     on_macos { assert_includes MachO::Tools.dylibs("testlibc++"), "#{opt_lib}/libc++.1.dylib" }
```

Comment:
Is it worth doing this here:
```suggestion
    assert_includes (testpath/"testlibc++").dynamically_linked_libraries, opt_lib/"libc++.1.dylib"
```

Reply:
It isn't always spelled this way, and the spelling is *technically* stable, but didn't seem worth encoding in a test.

But I could replace this with code like below that does `/#{opt_lib}/libc\+\+/` in a match in a loop if you like?

Reply:
Ah, yes, whoops, that will need to be
```suggestion
    assert_includes (testpath/"testlibc++").dynamically_linked_libraries, opt_lib/shared_library("libc++", "1")
```